### ISSUE TITLE
Can meeting minutes be a separate blog?

### ISSUE THREAD

**Issue by wesholing** [eastbaydsa/website#89]
On the current NationBuilder site, I enter meeting minutes as blog posts so that they automatically get entered into a blogroll.
https://www.eastbaydsa.org/resources-minutes

Can we have more than one blog on the new site? If this is a problem or a lot of work, I can just leave them as pages.

**Comment by dominic** [comment]
Yep - we can do this! For now, can they be in the main blog roll? It'll be easy to split them in the future.

**Comment by wesholing** [comment]
If we did that, would they also be listed among the blog posts? I'm not so interested in that, more the functionality of having a blogroll like on the current site. If not, we can just leave it as-is.

**Comment by dominic** [comment]
Right now they would be. We would split them up in the future; I'd just enter them all as blog posts today and we'll filter them into specific blogrolls in the future.

**Comment by wesholing** [comment]
Ah, okay. We can just leave this as-is for now then. Definitely not
anything that needs to be done before launch.

**Comment by dominic** [comment]
FWIW it's actually gonna be fairly easy. We could certainly prioritize it before launch.

**Comment by tilgovi** [comment]
Bump. Should we do this? What's happening with meeting minutes currently?

**Comment by wesholing** [comment]
+Mark Gabriel <mjpgabriel@gmail.com>

Thanks for following up, Randall. Currently, meeting minutes are added to
the site like any other page, then linked manually from one of the minutes
pages (general meetings <https://www.eastbaydsa.org/meetings/minutes>
or Steering
Committee meetings <https://www.eastbaydsa.org/steering-committee/minutes>).

I'd filed this ticket originally to eliminate that last step by treating
meeting minutes similarly to how we treat blog posts, appearing dynamically
on the site as a list, but Mark now handles production of minutes. Mark,
would this be a useful feature, or should we just cancel the request?

Wes
